api/src/OAuth/HitobitoUser.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\OAuth;
+
+use League\OAuth2\Client\Provider\ResourceOwnerInterface;
+
+class HitobitoUser implements ResourceOwnerInterface {
+    /**
+     * @var array
+     */
+    protected $response;
+
+    public function __construct(array $response) {
+        $this->response = $response;
+    }
+
+    public function getId() {
+        return $this->response['id'];
+    }
+
+    /**
+     * Get preferred display name.
+     */
+    public function getName(): string {
+        return $this->response['name'];
```

Comment:
In der Response von hitobito ist `name` nie drin soweit ich das im Code von hitobito sehe... Ist das egal?

Reply:
Das macht keinen Sinn, merci